frontend/src/pages/Notifications/actions.js

```diff
@@ -105,26 +103,18 @@ export function fetchHistoryItems(project, offset, limit) {
   };
 }
 
-export function markMultipleNotificationsAsRead(notificationIds, offset, limit) {
+export function markMultipleNotificationsAsRead(notificationIds, notificationPage) {
   return {
-    type: MARK_MULTIPLE_NOTIFICATION_AS_READ,
+    type: MARK_MULTIPLE_NOTIFICATIONS_AS_READ,
```

Comment:
How do we know how big the page is? (25 or 50)?

Reply:
and why do we need to send `page` and `ids`

Reply:
- the page size is saved in the state
- we send the page to reload after the messages have been marked as read
- we send the IDs because they are already available in the component. if we would send the page only, we would have to make another call to the API to get all the messages from the current page to extract the IDs, which would slow down the process.